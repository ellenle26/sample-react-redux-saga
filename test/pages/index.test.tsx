/**
 * @jest-environment jsdom
*/

import {render} from "@testing-library/react";
import React from "react";

import Home from "../../src/pages/index";

test("snapshot testing", () => {
  const renderResult = render(<Home />);
  expect(renderResult.asFragment()).toMatchSnapshot();
});