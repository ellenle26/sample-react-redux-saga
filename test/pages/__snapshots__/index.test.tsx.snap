// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="p-4"
  >
    Porters
  </div>
</DocumentFragment>
`;
